### PR TITLE
Added Meeting Notes from Notion

### PR DESCRIPTION
**Pull request by Darin8**
Exported each document from the meetings from Notion as markdown files.

Let me know if there are any other documents needed from Notion.

### CODE REVIEW

File: admin-cse112/meeting/Sprint 1 Review.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- keeping a good working pace
+- Started strong and continuing
+- Updates on whats happening
+- DevOps was quickly built
+- Everyone is doing stuff
+- we know what each of us should be doing
+- going to professor Powell and Sanat (our stakeholders)
+- project board is easy to see what people are working on
+- Everyone has been working on the project especially with covid breakout
+-
```
Comment (forschoolandnothingelse) [inline admin-cse112/meeting/Sprint 1 Review.md]: Remove this bullet point

File: admin-cse112/meeting/Week 5 Product Meeting.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+- Waynar and Darin integrated webhack so all js files and sent to server quickly and then the next problem is ssh proble ms. on pr merge this action will have the server run a git checkout to pull the.
+- git repo was made and people will be added to the org
+- Backend was left untouched, Front-end has a bundler that makes a product js code that will be much much faster.
+- Decided on using forking
+- two new branches which are back-end and front-end for pushing to make sure they run well
+- Linting has been done so that we can check with git-hub actions running ‘lint’ should run the check and fixing it can be done with a command that Waynar brought up.
+- Initial guideline doc has been made for making testing tools
+- James is going to deal with back-end testing whereas Richard will be doing front-end testing. By that this means they will be working closely with that team in order to keep them in check.
+-
```
Comment (forschoolandnothingelse) [inline admin-cse112/meeting/Week 5 Product Meeting.md]: Remove this bullet

File: admin-cse112/meeting/Week 5 Product Meeting.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Clone the repo
+- Run npm install in your portion of the repo
+- Do not upload the npm folder dependencies.
+- BEFORE you push to the repo, make sure to run the build so we have all the necessary resources. `npm run build` should be run
+
+# Potential blockers?
+
+- Understanding the Repo
+
+# Action items
```
Comment (forschoolandnothingelse) [inline admin-cse112/meeting/Week 5 Product Meeting.md]: Remove this section

File: admin-cse112/meeting/Week 6 Product Meeting.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+- [x]  Progress on the Sprint
+- [x]  Discussing the Domain
+- [x]  UI and reshaping the Sprint
+
+# What did we do last week?
+
+- We worked on our tasking
+
+# What are we doing this week?
```
Comment (forschoolandnothingelse) [inline admin-cse112/meeting/Week 6 Product Meeting.md]: Remove this empty section

File: admin-cse112/meeting/Week 6 Product Meeting.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+- 
+
+# Potential blockers?
+
+-
```
Comment (forschoolandnothingelse) [inline admin-cse112/meeting/Week 6 Product Meeting.md]: Add "none"

File: admin-cse112/meeting/Week 7 Product Meeting.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+Created: May 13, 2022 3:43 PM
+Last Edited Time: May 13, 2022 4:30 PM
+Participants: Anonymous, Anonymous, Anonymous, Anonymous, Anonymous, Anonymous, Anonymous, Anonymous
+Type: Product Meeting
+
+# Agenda
+
+- [ ]  Discuss Professor Powell’s suggested changes
+
+# What did we do last week?
```
Comment (forschoolandnothingelse) [inline admin-cse112/meeting/Week 7 Product Meeting.md]: Remove this

File: admin-cse112/meeting/Week 7 Product Meeting.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+- 
+
+# What are we doing this week?
```
Comment (forschoolandnothingelse) [inline admin-cse112/meeting/Week 7 Product Meeting.md]: Remove this

File: admin-cse112/meeting/Week 8 Product Meeting.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
+# Week 8 Product Meeting
+
+Created: May 20, 2022 3:45 PM
+Last Edited Time: May 20, 2022 4:17 PM
+Type: Product Meeting
+
+# Agenda
```
Comment (forschoolandnothingelse) [inline admin-cse112/meeting/Week 8 Product Meeting.md]: Remove this section

File: admin-cse112/meeting/Week 8 Product Meeting.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+- Testing tasks
+- new frontend log
+- testing the backend tests
+- modifying backend to reflect new pages.
+
+# Potential blockers?
+
+- None spoken of
+
+# Action items
```
Comment (forschoolandnothingelse) [inline admin-cse112/meeting/Week 8 Product Meeting.md]: Remove this section

File: admin-cse112/meeting/Week 9 Product Meeting.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+- [ ]  Draft Jest general testing guideline
+- [ ]  Delete .vscode
+- [ ]  Finish Jest testing for delete functions
+- [ ]  Components with the right sidebar
+- [ ]  Join daily Zoom rooms to work on your stuff and let teammates know
+- [ ]  Need more hands to tackle tasks (small amount of people working on multiple tasks)
+- [ ]  Fix bugs that arise from next week
+
+# Meeting Notes
```
Comment (forschoolandnothingelse) [inline admin-cse112/meeting/Week 9 Product Meeting.md]: Remove this section

**Review by forschoolandnothingelse — CHANGES_REQUESTED** [review]

**Review by e5chang — APPROVED** [review]
I agree about deleting any empty sections and keep useful/relevant comments. Otherwise looks good

**Review by waynarbocangel — APPROVED** [review]

**Review by forschoolandnothingelse — APPROVED** [review]
Thanks for making those changes.

**Pull request merged**